### DISCUSSION TITLE
Unclear how to go from IActorQueryOperationOutput to Sparql Result JSON

### DISCUSSION THREAD

**Discussion by michielbdejong** [comunica/comunica#452] [Q&A] [answered]
Given an `IActorQueryOperationOutput`, is there also a way to get that result in the https://www.w3.org/TR/sparql11-results-json/ format?

Is there a collection of examples of somewhere that I missed, or any way in which I can find information like this myself?

**Comment by rubensworks** [comment, marked as answer]
That's not documented (yet) indeed, but that's definitely possible.

Your query engine should have a [`.resultToString(queryResult)` method](https://github.com/comunica/comunica/blob/master/packages/actor-init-sparql/lib/ActorInitSparql-browser.ts#L168-L195), that by default should be able to serialize to `application/json` according to the SPARQL 1.1 JSON Results spec.
